casanovo/casanovo.py

```
@@ -505,7 +509,7 @@ def setup_model(
     return config, model
 
 
-def _get_model_weights(cache_dir: Path) -> Path:
+def _get_model_weights(cache_dir: Path, is_timstof: bool) -> Path:
```

Ruff (0.15.9)

[warning] Boolean-typed positional argument in function definition

(FBT001)